resources/init.groovy.d/global-args-security.groovy

```diff
@@ -32,6 +34,14 @@ def qpsPipelineGitBranch = env['QPS_PIPELINE_GIT_BRANCH']
 
 def qpsPipelineLogLevel = env['QPS_PIPELINE_LOG_LEVEL']
 
+// def sonarName = env['SONAR_NAME']
```

Comment:
1. I don't see sonar related vars in Dockerfile.
2. sonarServerUrl using your value doesn't have protocol, i.e. "demo.qaprosoft.com/sonarqube".
Anyway in this place there is no sense to concatenate string. Just dclare new varibale: SONAR_URL
We could generate url using QPS_HOST on docker-composer.yml or variables.env layer.
3. I don't see any reference to sonarCredentialsId and others in the code

Reply:
env vars added in  [3ed95d2](https://github.com/qaprosoft/jenkins-master/pull/200/commits/3ed95d2cbf1f9140c573951a03ae157ad7bf1be8)